### ISSUE TITLE
Feature request: Ability to sync tabs with other devices

### ISSUE THREAD

**Issue by rodude123** [projectdelphai/panorama-tab-groups#199]
From before you can import tabs which is great but if there was some way to sync the tabs with other devices such as from desktop to a laptop would be great. I don't mind implementing this idea. I have some ideas about how to do it. There could be a button to sync with OneDrive or Google Drive for example, and it updates it based on how frequently they want to update it e.g. every minute or every 5 minutes. Also allowing updating it when the browser closes no matter what.. The last one might be a bit hard but I'll see how far I get

**Comment by tech189** [comment]
Yes, this would be such a useful feature! OneDrive or Google Drive would be okay but if it could piggyback on Firefox's built-in sync that would be very nice, if at all possible - maybe like how uBlock Origin syncs its settings and custom rules

**Comment by rodude123** [comment]
It's been quite a while and no one has said anything yet so I'm going to try and see if I can work to bring this feature to life

**Comment by kaanaksit** [comment]
I can't agree more. Panorama is a great add-on. Thank you for making it possible!

Does anyone know where do panorama store the list of tabs in the local? I mean is there a way to copy a list to an another device the hard way?

**Comment by rodude123** [comment]
I think syncing to firefox sync is now possible with this https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/set. I might have a go at this and see if it is possible.
